components/scream/src/physics/rrtmgp/mo_garand_atmos_io.cpp

```diff
@@ -0,0 +1,115 @@
+#include "mo_garand_atmos_io.h"
+#include "simple_netcdf.hpp"
+#include <netcdf.h>
+
+// Read in the data, then use only the first column, and copy it to all of the model columns
+// In the end, all model columns will be identical
+void read_atmos(std::string input_file, real2d &p_lay, real2d &t_lay, real2d &p_lev, real2d &t_lev,
+                GasConcs &gas_concs, real2d &col_dry, int ncol) {
+  simple_netcdf::SimpleNetCDF io;
+  io.open(input_file , NC_NOWRITE);
+
+  int nlay = io.getDimSize("lay");
+  int nlev = io.getDimSize("lev");
+
+  p_lay = real2d("p_lay",ncol,nlay);
+  t_lay = real2d("t_lay",ncol,nlay);
+  p_lev = real2d("p_lev",ncol,nlev);
+  t_lev = real2d("t_lev",ncol,nlev);
+
+  real2d tmp2d;
```

Comment:
This is a reflection of my own ignorance: is this a 2d array of reals that uses 1-based indexing?

Reply:
Yes. It was decided to keep the 1-based indexing in the port of the RRTMGP code from Fortran to C++ to simplify the process. Kind of awkward, but is what it is.